app/aws/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+desc: 
+file: __init__.py.py
```

Comment:
I like the practice to always have an __init__.py in every module. I know it is required for python2, and python3 allows modules don't come with it. Just curious what are more advantages of __init__.py?